resources/definitions/strateo3d_IDEX420.def.json

```diff
@@ -0,0 +1,132 @@
+{
+    "version": 2,
+    "name": "IDEX420",
+    "inherits": "fdmprinter",
+    "metadata":
+        {
+            "author": "eMotionTech",
+            "manufacturer": "eMotionTech",
+            "visible": true,
+            "file_formats": "text/x-gcode",
+            "has_machine_quality": true,
+            "has_materials": true,
+            "has_variants": true,
+            "preferred_variant_name": "IDEX420 Standard 0.6",
+            "preferred_material": "emotiontech_pla",
+            "preferred_quality_type": "c",
+            "variants_name": "Print Head",
+            "machine_extruder_trains": 
+                { 
+                    "0": "strateo3d_IDEX420_left_extruder", 
+                    "1": "strateo3d_IDEX420_right_extruder"
+                }
+        },
+        
+    "overrides": 
+        {
+            "machine_name": { "default_value": "IDEX420" },
+			"machine_start_gcode": { "default_value": "G90 ; switch to absolute coordinate mode\nT{initial_extruder_nr} ; select the firstly used tool\nG1 F18000 Y-160 Z15 ; move fast to the coordinates\nG1 F18000 X0 Z0.3 ; move fast to the coordinates\nG92 E0 ; set the extruder to 0\nG1 F300 X60 E24 ; purge the hotend\nG1 F600 X40 ; swipe the nozzle\nG1 F600 Z3 ; perform Z hop" },
```

Comment:
Please use consistent spacing / tabs here. The identation is a bit messy right now.

Reply:
Hello nallath, Sorry for indentation, I developp on VScode on windows so I didn't saw this error. I redo the indentation and open the file with notepad++, i doesn't seem to be mess anymore.